packages/pages/src/components/address/methods.ts

```diff
@@ -22,3 +27,152 @@ export const getUnabbreviated = (
 
   return unabbreviatedField && address[unabbreviatedField];
 };
+
+/**
+ * Get a third-party maps url for a Yext location
+ *
+ * @param {any} profile - Partial of Yext Location entity profile
```

Comment:
I feel like we should specify the profile fields we require?

Reply:
Agreed. This should take in the items it needs rather than the entire profile.